docs/app/components/ButtonDocs/example.txt

```diff
@@ -0,0 +1,4 @@
+<Button
+  buttonStyle="primary"
+  text="Primary Button"
+  onClick={() => {}}/>
```

Comment:
Lets remove the onClick function here since it's not being used on the component. Seems confusing to pass an empty function